packages/nc-gui/composables/useSharedView.ts

```diff
@@ -1,13 +1,16 @@
 import type { ExportTypes, FilterType, PaginatedType, SortType, TableType, ViewType } from 'nocodb-sdk'
 import { UITypes } from 'nocodb-sdk'
 import { useNuxtApp } from '#app'
+import { useGlobal } from '#imports'
 
 export function useSharedView() {
   const nestedFilters = useState<(FilterType & { status?: 'update' | 'delete' | 'create'; parentId?: string })[]>(
     'nestedFilters',
     () => [],
   )
-  const paginationData = useState<PaginatedType>('paginationData', () => ({ page: 1, pageSize: 25 }))
+  const { appInfo } = $(useGlobal())
+  const appInfoDefaultLimit = appInfo?.defaultLimit || 25
```

Comment:
Unnecessary `?` - appInfo always has *some* value, it's never supposed to be undefined.